Accounts code examples

gabrielkuettel opened the pull request:
Completes:

DVP-534
DVP-536
DVP-548
DVP-552
DVP-553

gabrielkuettel committed:
feat: add account code examples

iskysun96 commented on projects/typescript-examples/algokit-utils-ts/accounts/creating-accounts.ts:
Add kmd account creation methods

gabrielkuettel replied:
must've added those since this comment :) 

```
  // example: KMD_WALLET_OPERATIONS
  /**
   * Create a wallet with the KMD client.
   */
  await algorand.client.kmd.createWallet('ACCOUNT_NAME', 'password')

  /**
   * Rename a wallet with the KMD client.
   */
  await algorand.client.kmd.renameWallet('ACCOUNT_NAME', 'password', 'NEW_ACCOUNT_NAME')
  // example: KMD_WALLET_OPERATIONS
  ```

iskysun96 commented on projects/typescript-examples/algokit-utils-ts/accounts/funding-accounts.ts:
we should show all of the different ways to get a dispenser. so add the localDispenser method.

there are also other ensure funded methods we can show

gabrielkuettel replied:
which one's missing? 

```
  // example: LOCALNET_DISPENSER
  /**
   * Get the default LocalNet dispenser account that can be used to fund other accounts
   */
  const localNetDispenser = await algorand.account.localNetDispenser()
  // example: LOCALNET_DISPENSER
  ```

iskysun96 commented on projects/typescript-examples/algokit-utils-ts/accounts/keys-and-signing.ts:
add example of building an unsigned txn and then manually sign it in the `newGroup().addTransaction()` method

gabrielkuettel replied:
added [40199be](https://github.com/algorandfoundation/devportal-code-examples/pull/10/commits/40199be76a8681267047e117040b0f0b37d10a24)

iskysun96 commented on projects/typescript-examples/algokit-utils-ts/accounts/multisignature-accounts.ts:
I think we should add examples sending payment transactions with the multisignature account.

gabrielkuettel replied:
wouldn't that be in scope for `transactions` and not `accounts`?

iskysun96 replied:
makes sense. I included in mine for now but we can circle back to this when we have more context on the pages

gabrielkuettel committed:
update creating account examples

gabrielkuettel committed:
add path aliases

gabrielkuettel committed:
add setup script

gabrielkuettel committed:
add account code examples

gabrielkuettel committed:
reorg project structure to ensure commands run

gabrielkuettel committed:
Merge branch 'main' into feat/DVP-367-add-ts-examples

gabrielkuettel committed:
chore: add eslint config; fix linting errors

gabrielkuettel committed:
test(smart contracts): add unit and integration tests for contracts

gabrielkuettel committed:
ci: add tealer installation to workflow

gabrielkuettel committed:
ci: add snapshots to pass ci-teal-diff

gabrielkuettel committed:
fix: directory name in algokit config

gabrielkuettel committed:
fix: resolve ci-teal-diff GitHub Action failures

The workflow was attempting to run TEAL diff checks concurrently across all
projects, causing Git lock conflicts. Split the checks into separate steps
with explicit working directories to run them sequentially. This ensures each
project's TEAL artifacts are validated independently and prevents Git lock
file collisions during CI runs.

gabrielkuettel committed:
ci: setup pre-commit hooks

gabrielkuettel committed:
chore: update lint and formatting configs

gabrielkuettel committed:
chore: update prettier ignore

gabrielkuettel committed:
chore: update pre-commit hooks for python stuff

gabrielkuettel committed:
chore: format code

gabrielkuettel committed:
feat(transactions): add override signer example

gabrielkuettel committed:
fix: wrong algo import
